### PR TITLE
BLOCKED - update the farmer info

### PR DESCRIPTION
**Pull request by Rexben001**
# Description

Refactor the add farmer inputs to be consistent with the data from the backend

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

**Comment by Pav0l** [comment]
[Fix 'Major Source Of Income’ and ‘Minor Source Of Income’ in Farmer Display](https://trello.com/c/GsIk5k27/89-fix-major-source-of-income-and-minor-source-of-income-in-farmer-display)

File: src/components/common/Input/addFarmerData.js [hunk 1]
```diff
@@ -234,25 +234,25 @@ export const personalInfo = {
     labelName: 'Nearest Landmark',
     value: '',
     valid: false,
-    touched: false,
+    touched: false
   },
-  Phone_1: {
```
Comment (quansenB) [inline src/components/common/Input/addFarmerData.js]: This change will likely break the display farmer page, If you ant to change it also change it in display famrmer to lower case

File: src/components/common/Input/addFarmerData.js [hunk 2]
```diff
@@ -234,25 +234,25 @@ export const personalInfo = {
...
-    labelName: ' Phone 1',
+    name: 'phone_1',
+    type: 'text',
+    labelName: ' phone 1',
     value: '',
     valid: false,
-    touched: false,
+    touched: false
   },
-  Phone_2: {
```
Comment (quansenB) [inline src/components/common/Input/addFarmerData.js]: Same issue as above

File: src/components/common/Input/addFarmerData.js [hunk 3]
```diff
@@ -371,51 +371,51 @@ export const familyInfo = {
...
     valid: false,
-    touched: false,
+    touched: false
   },
-  highest_level_of_dependent: {
+  highest_level_of_dependants: {
     elementType: 'select',
-    name: 'highest_level_of_dependent',
+    name: 'highest_level_of_dependants',
     labelName: 'Highest Level Of Education Level',
```
Comment (quansenB) [inline src/components/common/Input/addFarmerData.js]: This label name makes no sense

**Pull request closed without merging**